server/liveblog/blogs/schema.py

```diff
@@ -41,6 +41,9 @@
     'public_url': {
         'type': 'string'
     },
+    'public_urls': {
```

Comment:
Is there any need to store all the urls in the schema? This seems like a recipe for bugs as you are effectively storing the data in two different places - once in the outputs and once here.